spec/features/groups/group_freezing_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.feature 'Group locking' do
+  context 'suite size still available, full group' do
+    it 'can be initiated by a leader' do
+      group = FactoryGirl.create(:full_group, size: 2)
+      log_in group.leader
+      visit draw_group_path(group.draw, group)
+      click_on 'Finalize Group'
+      expect(group.locked_members).to eq([group.leader])
```

Comment:
Isn't this dual expectation going to fail rubocop? Maybe turn it into separate method?